libnetwork/docs/legacy.md

```diff
@@ -1,5 +1,5 @@
 
-This document provides a TLD&R version of https://docs.docker.com/v1.6/articles/networking/.
+This document provides a TL;DR version of https://github.com/moby/moby/blob/v1.6.0/docs/sources/articles/networking.md.
```

Comment:
Seems a quite old version.
Maybe we can just remove `legacy.md`?

Reply:
Yeah, so this page looks to be intended to show "what it worked like before libnetwork". We should definitely clean up some of those docs, but I thought "let me update it once more to keep it functional, and clean up later (next release?)